Psr/Log/Test/LoggerInterfaceTest.php

```diff
@@ -2,16 +2,21 @@
 
 namespace Psr\Log\Test;
 
+if (!class_exists('\\PHPUnit_Framework_TestCase', true)) {
```

Comment:
This does not make sense. You are creating an alias only when the **old** name does not exist (i.e. in PHPUnit 6+) while you use only the new name.

Reply:
Sorry, my bad...
I fixed it. Could you review it again?